scripts/mdpow-solvationenergy

```diff
@@ -185,31 +226,48 @@ if __name__ == "__main__":
     parser.add_option('-p', '--plotfile', dest="plotfile",
                       help="plot dV/dlambda to FILE; use png or pdf suffix to "
                       "determine the file type. 'auto' generates a pdf file "
-                      "DIRECTORY/dVdl_<molname>_<solvent>.pdf and 'None' disables it [%default]",
+                      "DIRECTORY/dVdl_MOLID_pow.pdf and 'None' disables it [%default]"
+                      "The plot function is only available for mdpow estimator,"
+                      "and is disabled when using alchemlyb estimators.",
+                      metavar="FILE")
+    parser.add_option('--solvent', '-S', dest='solvent',
+                      metavar="NAME",
+                      help="solvent NAME for compound, 'water', 'octanol', or 'cyclohexane' [%default]")
+    parser.add_option('-o', '--outfile', dest="outfile",
+                      help="append one-line results summary to FILE [%default]",
                       metavar="FILE")
     parser.add_option('-e', '--energies', dest="energyfile",
                       help="append solvation free energies to FILE [%default]",
                       metavar="FILE")
+    parser.add_option('--estimator', dest="estimator", default='alchemlyb',
+                      help="choose the estimator to be used, alchemlyb or mdpow estimators")
+    parser.add_option('--method', dest="method", default='MBAR',
+                      help="choose the method to calculate free energy",)
     parser.add_option('--force', dest='force',
                       action="store_true",
                       help="force rereading all data [%default]")
+    parser.add_option('--noSI', dest='SI',
+                      action="store_false",
+                      help="Disable statistical inefficiency analysis"
+                      "Statitical inefficiency analysis is performed by default when using"
+                      "alchemlyb estimators and is disabled when using mdpow estimator.")
```

Comment:
make this a [BooleanOptionalAction action option with argparse](https://docs.python.org/3/library/argparse.html#action) so that we have `--SI` and `--no-SI` (helps with #147 )